### PR TITLE
Bug/d3 asim 1525

### PR DESCRIPTION
**Pull request by hannesdiedrich**

### CODE REVIEW

File: src/d3a/models/strategy/area_agents/two_sided_pay_as_clear_engine.py [hunk 1]
```diff
@@ -180,7 +179,6 @@ def _accept_cleared_offers(self, clearing_rate, clearing_energy, accepted_bids):
                 break
             elif (math.floor(offer.price / offer.energy)) <= clearing_rate and \
                     (clearing_energy - cumulative_traded_offers) >= offer.energy:
-                # TODO: Used the clearing_rate as the original_trade_rate for the offers, because
```
Comment (spyrostz) [inline src/d3a/models/strategy/area_agents/two_sided_pay_as_clear_engine.py]: This TODO should not be deleted IMO

Comment (hannesdiedrich) [reply src/d3a/models/strategy/area_agents/two_sided_pay_as_clear_engine.py]: Oh yes! This was not on purpose. I had my own little TODO there and deleted both. Will re-add again.

File: integration_tests/jira_issues_tests.feature [hunk 2]
```diff
@@ -70,14 +70,19 @@ Feature: Jira Issues Tests
...
-   Scenario: D3ASIM-1475, default_2off finishes successfully for two-sided pay as clear market
-     Given we have a scenario named jira/default_2off_d3asim_1475
-     And d3a is installed
-     Then we run the simulation with setup file jira.default_2off_d3asim_1475 and parameters [24, 60, 60, 0, 1]
+  Scenario: D3ASIM-1475, default_2off finishes successfully for two-sided pay as clear market
+    Given we have a scenario named jira/default_2off_d3asim_1475
+    And d3a is installed
+    Then we run the simulation with setup file jira.default_2off_d3asim_1475 and parameters [24, 60, 60, 0, 1]
+
+  Scenario: D3ASIM-1515, PAC simulation finishes without assert in PAC engine
```
Comment (faizan2590) [inline integration_tests/jira_issues_tests.feature]: Very minor, but I guess you meant `D3ASIM-1515..`

Comment (hannesdiedrich) [reply integration_tests/jira_issues_tests.feature]: Yes, you are right! Will change this.

**Pull request merged**